src/test/javascript/spec/component/competencies/prerequisite-form.component.spec.ts

```diff
@@ -97,6 +97,7 @@ describe('PrerequisiteFormComponent', () => {
         prerequisiteFormComponentFixture.whenStable().then(() => {
             expect(submitFormSpy).toHaveBeenCalledOnce();
             expect(submitFormEventSpy).toHaveBeenCalledOnce();
+            discardPeriodicTasks();
         });
```

Comment:
I don't think this part is actually being executed, which is why it still fails. Do we really need the `whenStable` here?

```suggestion
        flush();
        expect(submitFormSpy).toHaveBeenCalledOnce();
        expect(submitFormEventSpy).toHaveBeenCalledOnce();
        discardPeriodicTasks();
```